src/components/KyberAITokenBanner/index.tsx

```diff
@@ -0,0 +1,435 @@
+import { ChainId, Currency } from '@kyberswap/ks-sdk-core'
+import { Trans } from '@lingui/macro'
+import { transparentize } from 'polished'
+import { memo, useMemo } from 'react'
+import { ArrowRight } from 'react-feather'
+import { useNavigate } from 'react-router'
+import { useMedia } from 'react-use'
+import { Text } from 'rebass'
+import styled, { keyframes } from 'styled-components'
+
+import bannerBackground from 'assets/images/truesight-v2/banner-background.png'
+import Column from 'components/Column'
+import ApeIcon from 'components/Icons/ApeIcon'
+import Row, { RowBetween, RowFit } from 'components/Row'
+import { APP_PATHS } from 'constants/index'
+import { NETWORKS_INFO } from 'constants/networks'
+import { NativeCurrencies } from 'constants/tokens'
+import { useActiveWeb3React } from 'hooks'
+import useMixpanel, { MIXPANEL_TYPE } from 'hooks/useMixpanel'
+import useTheme from 'hooks/useTheme'
+import KyberScoreMeter from 'pages/TrueSightV2/components/KyberScoreMeter'
+import { NETWORK_TO_CHAINID } from 'pages/TrueSightV2/constants'
+import { SUPPORTED_NETWORK_KYBERAI } from 'pages/TrueSightV2/constants/index'
+import { useTokenDetailQuery } from 'pages/TrueSightV2/hooks/useKyberAIData'
+import { calculateValueToColor } from 'pages/TrueSightV2/utils'
+import { useIsWhiteListKyberAI } from 'state/user/hooks'
+import { MEDIA_WIDTHS } from 'theme'
+
+const StaticSupportedToken: {

```

Comment:
use `NativeCurrencies`

Reply:
I think this is more flexible to add new tokens for this banner if have update request